2_horseshoe_bat/api_definitions/rest/metadata_repository.yaml

```diff
@@ -0,0 +1,2404 @@
+          title: Creation Date
+          type: string
+        description:
+          description: A detailed description (abstract) that describes the goals
+            of             this Study.
+          title: Description
+          type: string
+        has_analysis:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/Analysis'
+            type: array
+          description: One or more Analysis entities associated with this Study.
+          title: Has Analysis
+        has_attribute:
+          description: 'Custom key/value pairs that further characterizes the Study.             (e.g.:
+            approaches - single-cell, bulk etc)'
+          items:
+            $ref: '#/components/schemas/Attribute'
+          title: Has Attribute
+          type: array
+        has_experiment:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/Experiment'
+            type: array
+          description: One or more Experiment entities associated with this Study.
+          title: Has Experiment
+        has_project:
+          anyOf:
+          - type: string
+          - $ref: '#/components/schemas/Project'
+          description: The project associated with this Study.
+          title: Has Project
+        has_publication:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/Publication'
+            type: array
+          description: One or more Publication entities associated with this Study.
+          title: Has Publication
+        id:
+          description: An identifier that uniquely represents an entity.
+          title: Id
+          type: string
+        release_date:
+          description: The timestamp (in ISO 8601 format) when the entity was released             for
+            public consumption.
+          title: Release Date
+          type: string
+        status:
+          allOf:
+          - $ref: '#/components/schemas/StatusEnum'
+          description: The status of a Study. For example, 'released' or 'unreleased'.
+        title:
+          description: Comprehensive title for the study.
+          title: Title
+          type: string
+        type:
+          description: The type of the Study.
+          title: Type
+          type: string
+        update_date:
+          description: Timestamp (in ISO 8601 format) when the entity was updated.
+          title: Update Date
+          type: string
+        xref:
+          description: Alternate identifiers for an entity.
+          items:
+            type: string
+          title: Xref
+          type: array
+      title: Study
+      type: object
+    Submission:
+      description: 'A grouping entity that represents information about one or more
```

Comment:
Maybe we should create a version of this submission model that is specific for creating a new submission (posting a submission)?